o8g/Scripts/actions.py

```diff
@@ -638,13 +642,11 @@ def isEnemy(cards):
 
 def turnManagementOn(group, x=0, y=0):
     mute()
-    setGlobalVariable("Automation", "Turn")
-    clearHighlights(group)
```

Comment:
Is it safe to remove these? What were they doing? It seems to have to do something with Turnmanagement